steps/olm/operator.go

```diff
@@ -87,3 +90,25 @@ var _ = gauge.Step("Validate tektoninstallersets status", func() {
 var _ = gauge.Step("Validate tektoninstallersets names", func() {
 	k8s.ValidateTektonInstallersetNames(store.Clients())
 })
+
+var _ = gauge.Step("Check version of component <component>", func(component string) {
+	defaultVersion := os.Getenv(strings.ToUpper(component + "_version"))
+	tkn.AssertComponentVersion(defaultVersion, component)
+})
+
+var _ = gauge.Step("Check version of <osp>", func(osp string) {
```

Comment:
Do you actually need `<osp>`? It's used only once so maybe we could hard-code the string here